mmpose/models/backbones/poseformer.py

```diff
@@ -0,0 +1,312 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn as nn
+from mmcv.cnn import build_norm_layer
+from mmcv.cnn.bricks.registry import DROPOUT_LAYERS
+from mmcv.cnn.bricks.transformer import FFN
+from mmcv.runner.base_module import BaseModule
+
+from ..builder import BACKBONES
+from .base_backbone import BaseBackbone
+
+
+class MultiheadAttention(BaseModule):
```

Comment:
mmcv also has MultiheadAttention module. Can we use that one?

Reply:
This MultiheadAttention module is adapted from **mmcls**. Compared with the MultiheadAttention in **mmcv**, its counterpart in **mmcls** is more similar to the implementation in the official **PoseFormer** repository.

Reply:
I will give **mmcv** Transformer blocks a try. However, if the output and the model weights of the re-implemented **PoseFormer** based on **mmcv** Transformer blocks could not fit the official version of **PoseFormer**, keeping the **mmcls** Transformer modules might be a good choice.